blog/_posts/2016-11-08-RubyConfPt.md

```diff
@@ -0,0 +1,37 @@
+
+The topic we chose for our talk was "Falling into Rabbit Holes", because this seemed the quintessence of the whole Rails Girls Summer of Code experience. <a href="https://twitter.com/theatanzt">Thea</a>, being a part of <a href="https://twitter.com/loadtocode/">LoadtoCode</a> and <a href="https://twitter.com/MayarAlaa122">Mayar</a>, being a part of <a href="https://twitter.com/RubysSecretteam">Ruby's Secret</a> shared their individual <a href="https://twitter.com/rubyconfpt/status/791674914302492672">Rails Girls Summer of Code experience</a> with the audience. For a wrap up the other team members had their personal shoutouts: <a href="https://twitter.com/lila_luca">Marie</a> from <a href="https://twitter.com/loadtocode/">LoadtoCode</a>, made clear that, now being in the process of changing careers and becoming a programmer, she is not automatically <a href="https://twitter.com/ines_opcoelho/status/791677324253159424">a person with superpowers</a> and remembering everyone of the fact that everybody can learn it and this message should be spread instead. <a href="https://twitter.com/Nada_Ashraf96">Nada</a> from <a href="https://twitter.com/RubysSecretteam">Ruby's Secret</a> was presenting really big goals, for example to have <a href="https://twitter.com/railsgirlssoc/status/791682827158945792">100 female coders in Cairo within 1 year</a> - support this mission with mentioning <i>#1year100women</i>! As this goal wouldn't have been big enough already she also invited everyone to a Ruby Conference happening in Egypt in 2017. One goal the team from Egypt already achieved though and this is to have the travel expenses to Portugal funded by the crowd - check out the awesome <a href="https://player.vimeo.com/video/188672656">video</a> they made to ask for support!
+
+![RGSoC On Stage](/img/blog/2016/2016-11-08-RubyConfOnStage.jpg) <font color="grey"><small><i>The Rails Girls Summer of Code Women on Stage at the Ruby Conference in Portugal (Image: Picture taken by RubyConf Portugal )</i></small></font><br>
+
+The People behind the Conference, mainly being part of <a href="https://subvisual.co/">subvisual</a>, were very welcoming and caring - we were even invited to a super delicious dinner for the speakers, where we were lucky enough to share the table with awesome people like <a href="https://twitter.com/martinfowler">Martin Fowler</a>, <a href="https://twitter.com/hiro_asari">Hiro</a>, <a href="https://twitter.com/pat">Pat Allan</a> and <a href="https://twitter.com/PragTob">Tobias Pfeiffer</a>. The <a href="https://twitter.com/RubysSecretteam">Ruby's Secret</a> Team even got a <a href="https://twitter.com/tenderlove/status/791683611040817152">Selfie with Aaron Patterson</a>.
```

Comment:
small addition: maybe add Hiro's last name ("Asari") to this list as well, to make the list consistent?